custom-metrics-stackdriver-adapter/adapter.go

```diff
@@ -108,7 +112,8 @@ func (sa *StackdriverAdapter) makeProviderOrDie(o *stackdriverAdapterServerOptio
 		stackdriverService.BasePath = o.StackdriverEndpoint
 	}
 
-	gceConf, err := gceconfig.GetGceConfig()
+	gceConf, err := getGceConfig(o)
+
```

Comment:
super nit: no need for a newline between an error-generating statement and checking its error